packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -226,7 +226,8 @@ export default class ConfigParser {
                     log.warn(`No suite was found with name "${suiteName}"`)
                 }
                 if (Array.isArray(suite)) {
-                    suiteSpecs = suiteSpecs.concat(ConfigParser.getFilePaths(suite, undefined, this._pathService))
+                    // Not supporting hierarchical suites - return will always be string[]
```

Comment:
The return type from getFilePaths is (string | string[])[], which can't be assigned to suiteSpecs which is type string[].  However, since we are not proposing to introduce the same kind of hierarchy into the suite spec, we know the return value will always be string[], and so by asserting it, this will compile.  As I said before, I am new to TypeScript, so there may be a better way to approach this, please advise.

Reply:
@erwinheitzman Can we close out this conversation